grails-app/domain/org/pih/warehouse/requisition/Requisition.groovy

```diff
@@ -209,6 +210,12 @@ class Requisition implements Comparable<Requisition>, Serializable {
         approvalRequired(nullable: true)
     }
 
+    Comment getRecentComment() {
+        return comments?.sort({ a, b ->
+            b.dateCreated <=> a.dateCreated
+        }).find()
```

Comment:
wouldn't `.first()` fit better here?
I believe `find()` should take some closure as an argument? I'm surprised that it works without anything

Reply:
@drodzewicz I think you still need to have here a null-safety operator with `find()` too.